src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -1871,37 +1873,37 @@ private String getTabLabel(Tab tab)
 
    private Tab tabForName(String name)
    {
-      if (name.equalsIgnoreCase("history"))
+      if (name.equalsIgnoreCase(HISTORY_PANE))
          return Tab.History;
-      if (name.equalsIgnoreCase("files"))
+      if (name.equalsIgnoreCase(FILES_PANE))
          return Tab.Files;
-      if (name.equalsIgnoreCase("plots"))
+      if (name.equalsIgnoreCase(PLOTS_PANE))
          return Tab.Plots;
-      if (name.equalsIgnoreCase("packages"))
+      if (name.equalsIgnoreCase(PACKAGES_PANE))
          return Tab.Packages;
-      if (name.equalsIgnoreCase("help"))
+      if (name.equalsIgnoreCase(HELP_PANE))
          return Tab.Help;
-      if (name.equalsIgnoreCase("vcs"))
+      if (name.equalsIgnoreCase(VCS_PANE))
          return Tab.VCS;
-      if (name.equalsIgnoreCase("tutorial"))
+      if (name.equalsIgnoreCase(TUTORIAL_PANE))
          return Tab.Tutorial;
-      if (name.equalsIgnoreCase("build"))
+      if (name.equalsIgnoreCase(BUILD_PANE))
          return Tab.Build;
-      if (name.equalsIgnoreCase("presentation"))
+      if (name.equalsIgnoreCase(PRESENTATION_PANE))
          return Tab.Presentation;
-      if (name.equalsIgnoreCase("presentations"))
+      if (name.equalsIgnoreCase(PRESENTATIONS_PANE))
          return Tab.Presentations;
-      if (name.equalsIgnoreCase("environment"))
+      if (name.equalsIgnoreCase(ENVIRONMENT_PANE))
          return Tab.Environment;
-      if (name.equalsIgnoreCase("viewer"))
+      if (name.equalsIgnoreCase(VIEWER_PANE))
          return Tab.Viewer;
-      if (name.equalsIgnoreCase("connections"))
+      if (name.equalsIgnoreCase(CONNECTIONS_PANE))
          return Tab.Connections;
-      if (name.equalsIgnoreCase("source"))
+      if (name.equalsIgnoreCase(UserPrefsAccessor.Panes.QUADRANTS_SOURCE))
```

Comment:
Small question: why do we need the extra package names prefixes here, compared to the other entries? (Why not have a separate `SOURCE_PANE = "Source"` entry below?)

Reply:
I was just using existing constants we already had for the quadrants (tabset1, source, etc.), but agree it's not pretty (or consistent).